src/Cake.NuGet.Tests/Fixtures/NuGetPackageInstallerFixture.cs

```diff
@@ -61,12 +61,23 @@ public NuGetPackageInstaller CreateInstaller()
             return new NuGetPackageInstaller(FileSystem, Environment, ProcessRunner, ToolResolver, ContentResolver, Log, Config);
         }
 
+        public Install.NuGetPackageInstaller CreateInProcessInstaller()
+        {
+            return new Install.NuGetPackageInstaller(FileSystem, Environment, ContentResolver, Log, Config);
+        }
+
         public IReadOnlyCollection<IFile> Install()
         {
             var installer = CreateInstaller();
             return installer.Install(Package, PackageType, InstallPath);
         }
 
+        public IReadOnlyCollection<IFile> InProcessInstall()
+        {
+            var installer = CreateInProcessInstaller();
+            return installer.Install(Package, PackageType, InstallPath);
```

Comment:
I *think* this will actually install stuff to the local filesystem as file system is not mocked on `NuGet.Client` side.